plugins.ts

```diff
@@ -230,6 +230,8 @@ export const inject = (deps: PluginDeps) => {
 					'scaffoldProjectDir',
 				];
 
+				if (key === 'help') val = false;
```

Comment:
If we pass `help` during the getPluginInfo stage to a plugin, we'll get yargs help screen for the plugin. I'll create a different PR to disable the help menu when invoking a plugin directly.

Reply:
What do you mean?

Reply:
In a test project, try running this and you'll see what I mean:
```
node node_modules/\@taqueria/plugin-flextesa/index.js --help
```

We should disable the help menu in the yargs config of the SDK so that we can pass the `--help` argument from the CLI to the plugin.

Reply:
Why do we need to pass --help to the plugins though? What is a use case for this?

Reply:
Oh, that's a separate point - we have always passed the help arg to plugins. I had thought about stripping it out but that seemed like a bigger change, as it would require modifying one of our core types, RequestArgs. But setting it to 'false' for ensure ensures that the help menu isn't output by a plugin.

Reply:
Still a bit confused. How does this affect the user? I believe they can do `taq compile --help` after installing the LIGO plugin in this PR no? I tried and it worked.